### PR TITLE
line width 300

### PR DESCRIPTION
**Pull request by jdahlblom**

### CODE REVIEW

**Comment by charliefoxtwo** [comment]
You'll want to make sure you reformat all files under `lib/io` and `lib/modules`

File: .stylua.toml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+column_width = 300
+
 [sort_requires]
-enabled = true
-column_width = 300
\ No newline at end of file
+enabled = true
```
Comment (charliefoxtwo) [inline .stylua.toml:4]: did the trailing newline get deleted here?

**Comment by jdahlblom** [comment]
> You'll want to make sure you reformat all files under `lib/io` and `lib/modules`

I haven't touched .styluaignore? Don't understand.

File: .stylua.toml (hunk 1 above)
Comment (jdahlblom) [reply .stylua.toml:4]: OK will change

**Comment by charliefoxtwo** [comment]
> I haven't touched .styluaignore? Don't understand.

by default, stylua only changes the files you edit on save. So if you haven't edited a file, it won't update it. You'll have to manually go through and reformat the files affected by this style change. You can do that by opening the file, right clicking, and clicking Format Document (or by using the keyboard shortcut)
image

**Pull request merged**